tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/AutoInstrumentation/AWS/Kinesis/ContextPropagationTests.cs

```diff
@@ -75,7 +75,8 @@ public void InjectTraceIntoRecords_WithJsonString_AddsTraceContext()
 
         var proxy = request.DuckCast<IPutRecordsRequest>();
 
-        ContextPropagation.InjectTraceIntoRecords(proxy, new PropagationContext(_spanContext, baggage: null));
+        var scope = AwsKinesisCommon.CreateScope(Tracer.Instance, "PutRecords", SpanKinds.Producer, null, out var tags);
```

Comment:
Could we try to remove the `Tracer.Instance` accesses here with a mock? We typically don't want to access `Tracer.Instance` in tests as it causes flake and other issues usually.

I think this example should be applicable to this: https://github.com/DataDog/dd-trace-dotnet/blob/7544783d76c0b8ed1a60e50a930648a5bcf611e5/tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/AutoInstrumentation/AWS/DynamoDb/AwsDynamoDbCommonTests.cs#L148